ouroboros-consensus-cardano/ouroboros-consensus-cardano.cabal

```
@@ -124,13 +124,14 @@ library
     , cardano-data
     , cardano-ledger-allegra
     , cardano-ledger-alonzo         ^>=1.3
+    , cardano-ledger-api            ^>=1.2.2
```

You'll need to set the lower bound to 
```suggestion
    , cardano-ledger-api            ^>=1.3.0
```

It has just change don master

---

Thank you!